client/.eslintrc

```diff
@@ -27,6 +27,7 @@
     }
   },
   "rules": {
+    "class-methods-use-this": [0],
```

Comment:
Why was it necessary to overwrite by this rule?

Reply:
In our services classes we are not using anything like "this.parentId" neither we can make these class methods static. So in order to bypass the warning, we had to add this rule.